app/components/home/UserHome/body/AddDocument/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
**Best Practice**

Use Pascal case for component files and small letters when it's an entry point.